### ISSUE TITLE
Definition of standardized names

### ISSUE THREAD

**Issue by DerVogel2020** [corona-warn-app/cwa-documentation#699]
## Your Question

* Documentation File:
* Line / Paragraph:
* Question:
Is there a definition of standardized names? I couldn't find an answer neither in the faq nor on github. 
I'm asking because I've got 2 first names and they are separated in the certificate view by a < which seems strange to me.

**Comment by jucktnich** [comment]
The ">" is the normal separator between names. I somehow think, that the rules are those from the iata, but I'm not sure why I think so, or if this is true.

**Comment by dsarkar** [comment]
@DerVogel2020 @jucktnich I think you can find the definition here: https://www.icao.int/publications/pages/publication.aspx?docnum=9303

However, I will cross-check internally.

**Comment by jucktnich** [comment]
Oh iata and icao are to similar 😅😂

**Comment by MikeMcC399** [comment]
@dsarkar / @DerVogel2020 

You might also like to refer to https://github.com/ehn-dcc-development/ehn-dcc-schema/blob/release/1.3.0/DCC.Core.Types.schema.json

        "gnt": {
          "title": "Standardised forename",
          "description": "The forename(s) of the person, transliterated ICAO 9303",
          "type": "string",
          "pattern": "^[A-Z<]*$",
          "maxLength": 80,
          "examples": [
            "JIRINA<MARIA<ALENA"
          ]
        }

**Comment by DerVogel2020** [comment]
Thanks. In https://www.icao.int/publications/Documents/9303_p3_cons_en.pdf I found "If the primary or secondary identifiers have more than one name component, each component shall be separated by a single filler character (<)." and "Where a hyphen appears between two name components, it shall be represented in the MRZ by a single filler character (<). (i.e. hyphenated names shall be represented as separate components).
Example
VIZ: MARIE-ELISE
MRZ: MARIE<ELISE"

Closing now because the question has been answered.
